### DISCUSSION TITLE
LOCAL library

### DISCUSSION THREAD

**Discussion by unecc** [music-assistant/support#1868] [Q&A] [answered]
Hi i wanna add to my music assistant a directory smb. please how can i do?

![image](https://github.com/music-assistant/hass-music-assistant/assets/134554559/b551d526-0f8a-4927-9265-9598b5b7bb6b)

![image](https://github.com/music-assistant/hass-music-assistant/assets/134554559/eff10e9a-3949-4520-b6da-33c1664136e8)

**Comment by OzGav** [comment, marked as answer]
Please read our documentation which is pinned to the top of the discussion area https://github.com/orgs/music-assistant/discussions/820
